### PR TITLE
Merge Monitoring & Logs integrations

### PR DESCRIPTION
**Pull request by fulyauluturk**
I combined the services that both have Monitoring & Logs integration under a single page and placed them at the top-level:
      - Apache: integration/apache-integration.md
      - Elasticsearch: integration/elasticsearch-integration.md
      - JVM: integration/jvm-integration.md
      - Kubernetes: integration/kubernetes.md
      - MySQL: integration/mysql-integration.md
      - Nginx: integration/nginx-integration.md
      - Opensearch: integration/opensearch-integration.md
      - PostgreSQL: integration/postgresql-integration.md
      - RabbitMQ: integration/rabbitmq-integration.md
      - Solr: integration/solr-integration.md
      - Varnish Cache: integration/varnishcache-integration.md
    The ones we have only Monitoring or Logs integration are still the same, listed under Monitoring or Logs. I also kept the old pages and trimmed them, provided a link to the new pages.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -144,6 +144,17 @@ pages:
       - FAQ: synthetics/faq.md
     - Integrations:
       - Overview: integration/index.md
+      - Apache: integration/apache-integration.md
```
Comment (otisg) [inline mkdocs.yml:147]: @fulyauluturk if all these were added here, shouldn't the mkdocs.yml diff show these removed from Logs and Monitoring sections?

**Review by otisg — APPROVED** [review]
LGTM.  Just one comment/question.

File: mkdocs.yml (hunk 1 above)
Comment (fulyauluturk) [reply mkdocs.yml:147]: We said we won't be removing the old ones but trim the pages and link to the new page. That's why I didn't remove them from mkdocs.

Comment (otisg) [reply mkdocs.yml:147]: Ah, right.

**Pull request merged**